### PR TITLE
gif media

### PR DESCRIPTION
**Pull request by benny-conn**
Changes:

- **Added** new `GIFMedia` graph QL type and mapped GIF media's to it

### CODE REVIEW

File: _encrypted_deploy [hunk 1]
```diff
@@ -1,13 +1,14 @@
... G4oBqqBDxH5IhZb 8LBZEgsO9arTJBv apn0FesgzkkBWAc fK6cB7fVa7tWKmH KZMdKZGobz4bju9 9LqYnMbwJtqhfpT tDZlF3lDBSQnPuE 8Q3COxmohaIVUEZ 9mFaVfqtKsFiZ0h Zi04aNfHETa0OBr KMFy1Y5zHRpOjtQ 0sXL5gjDZrI6u6U LQt7K2UoDKRfuYe Hh0W5SgH6ja3fAA m1txKkGkeSo0WS0 2e7h5T08FUI1Rtd PPhA9SleQazRvrQ JiQ0leIVRIoFfer Ls48MadHpNwTRBa 6hayxYVjTQkWVu7 Tm10P19MQoSKiIa 5kvkDNV7JvXdlNI OsjlkpypZ90AnXJ yyKlP7vbqQcIKI4 sFGEueez6N1i5fz cszxBUNY8rMDlRs rDy5LcCnUTzYKne t5BCFAatbRSJnMj akG0Aa8n5MSGwER xf2VltubI6wYIx3 dXsa1nQwsgora5g 4hHE8UO89zsX4dl yPH7hPozJSyb1WD Fiw0Dt8QY0NvcJk AIJv4u01sl4QCoW vnczTMpWE74zLnB md4ILWhd5PA2ppu zcH6F9gGSrbZ1Bx sxkBukPXlKBrWGp taolnptFrAzdntx KHZ6Saqvc60sVAF 3r3Ls3CFEJXC99p rCVV725wXXY2Tz5 9LVu8XBez6MhGWs LF271Lcgpa3Vm86 pE6D3UtfNnsvSbv hSYRh5xH9yzJifu ZPcw33apfYxf5Db x0OWSOmhwM0QAtd 1A0A5jRWxcge4Lt fkXvTi622HTpPtQ fePTUDukWC9JlEu uAiHMHXAcyH3Kjr 3QDRoZvDB5wh55S GaDGUh0Qy4nrgz1 DSyQk8aXqklZE6a hqdx8yMhdhyYSwk jOI7KTE5MYf3FaY Gyo5XXafZ7RrOiu DFLFVivCZieiKJl vJ9xB9Dz4MSqdrV yyJ1LCyUCbeO8kz a0tHTpbVxa46Ehp iucjuudFcWAdW5W R3BTaM6N2LvbBtt dpYOmKt6EHiWplM MVzIgqOis3tsZaD sFxzNy2R9a7Y1Fa dHvQnLYxxSHyNo5 9VgJD0hCBqp8I0W OS78VXJcKhvRcWN RRhtxJOvZeHahPI 47CWTIuKIN8QKFV dp4hniq1tLJCTsq ug5QsBBMc77F95t
```
Comment (radazen) [inline _encrypted_deploy:1]: What changed in `_encrypted_deploy`?

Comment (benny-conn) [reply _encrypted_deploy:1]: I added the indexer server file that was necessary to deploy that wasn't there, totally unrelated, but there it is!

Comment (radazen) [reply _encrypted_deploy:1]: Makes sense!

**Review by Robinnnnn — APPROVED** [review]

File: _encrypted_deploy (hunk 1 above)
Comment (jarrel-b) [reply _encrypted_deploy:1]: ohh are those the `app-<env>-indexer-server.yaml` files? The service is on cloud run so it uses the `indexer-server-env.yaml` files unless those were missing too

**Pull request merged**